rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2024-10-13"
+channel = "1.83.0"
```

Comment:
@drewstone this always needs to be nightly for `rustdoc-types`, please make sure it stays in line with the version it needs.